js/ascendex.js

```diff
@@ -1289,7 +1289,10 @@ module.exports = class ascendex extends Exchange {
             market = this.market (symbol);
             request['symbol'] = market['id'];
         }
-        const method = this.safeValue (options, 'method', 'v1PrivateAccountGroupGetOrderHist');
+        let method = this.safeValue (options, 'method', 'v1PrivateAccountGroupGetOrderHist');
+        if (market['swap']) {
```

Comment:
The `market` is not always defined here.

Reply:
See how it's done here: https://github.com/ccxt/ccxt/blob/master/js/okex.js#L2062